en/docs/monitoring-and-insights/view-logs.md

```diff
@@ -0,0 +1,106 @@
+# View Logs
+
+The unified log view in Choreo allows you to gain application and user insights while ensuring data privacy. 
+
+Choreo provides real-time insights through live logs and allows you to view historical logs for insights into the past. You also have the flexibility to define a required time window to view relevant log entries, where an intuitive scrolling capability facilitates easy viewing of appropriate logs within the selected time frame.
+
+The log view also provides advanced filtering capabilities that allow you to efficiently navigate through appropriate logs to expedite troubleshooting in distributed environments.
+
+!!! info "Note"
+        All personally identifiable information (PII) gets resolved at the frontend service level, with only relevant UUIDs stored in logs.
+
+## Runtime logs
+
+Choreo runtime logs provide insights into project-level and component-level logs, covering application, gateway, system, and access logs. The runtime logs streamline the debugging process by centralizing diverse log sources.
+
+You can view runtime logs for the past 30 days by default when you access the runtime logs page. Choreo also allows precise filtering of runtime logs based on parameters such as log level (error, warn, info, debug), log type (application, gateway, system, access), and environment (development, staging, production). These granular filters effectively simplify the debugging process. 
```

Comment:
Shall we say who can view runtime logs? Sicne we talk about permissions for other type of logs.

Reply:
+1
Added a sentence on this.